numba/cuda/tests/cudadrv/test_nvvm_driver.py

```diff
@@ -15,7 +14,8 @@
 class TestNvvmDriver(unittest.TestCase):
     def get_nvvmir(self):
         if NVVM().is_nvvm70:
-            metadata = metadata_nvvm70
+            versions = NVVM().get_ir_version()
+            metadata = metadata_nvvm70 % versions
```

Comment:
Perhaps update to use `.format()` or an `f-string`?

Reply:
I decided against that because it makes the IR less legible due to the presence of `{` and `}`.